assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -297,6 +297,11 @@ spec:
           value: /healthz
         - name: HEALTHZ_PORT
           value: ":8081"
+        - name: HOST_IP
```

Comment:
This was added to the wrong container.  This is the KMS container not the Kubernetes API server container.